src/app/productConfig/productConfigForm/productConfigForm.tpl.html

```diff
@@ -156,200 +156,197 @@ <h4 class="panel-title border-bottom-small mt" translate>
         <div class="panel-body pt0" ng-if="$ctrl.useV3 !== 'true'">
           <div ng-if="$ctrl.useSuggestedAmounts">
             <div class="radio radio-custom-amount"
-                  ng-repeat="suggested in $ctrl.suggestedAmounts | orderBy:'order'">
+                 ng-repeat="suggested in $ctrl.suggestedAmounts | orderBy:'order'">
               <label>
                 <input name="suggestedAmount"
-                        type="radio"
-                        ng-click="$ctrl.changeAmount(suggested.amount)"
-                        ng-checked="!$ctrl.customInputActive && $ctrl.itemConfig.AMOUNT === suggested.amount" />
+                       type="radio"
+                       ng-click="$ctrl.changeAmount(suggested.amount)"
+                       ng-checked="!$ctrl.customInputActive && $ctrl.itemConfig.AMOUNT === suggested.amount" />
                 {{$ctrl.suggestedAmount(suggested.amount)}} {{suggested.label}}
               </label>
             </div>
             <div class="radio radio-custom-amount form-inline" ng-class="{'has-error': ($ctrl.itemConfigForm.amount | showErrors)}">
               <div class="form-group">
                 <label>
-                  <input 
-                    name="suggestedAmount"
-                    type="radio"
-                    ng-checked="$ctrl.customInputActive"
-                  />
-                  <input
-                    name="amount"
-                    class="form-control form-control-subtle"
-                    type="text"
-                    id="ca"
-                    step="1"
-                    tabindex="-1"
-                    ng-model="$ctrl.customAmount"
-                    ng-change="$ctrl.changeCustomAmount($ctrl.customAmount)"
-                    ng-focus="$ctrl.customInputActive = true;"
-                    ng-required="$ctrl.customInputActive"
-                    placeholder="{{'OTHER_PLACEHOLDER' | translate}}" />
+                  <input name="suggestedAmount"
```

Comment:
These changes aren't related to ng-if and should be squashed into the previous commit.